controllers/machinehealthcheck_controller.go

```diff
@@ -85,20 +85,20 @@ func (r *MachineHealthCheckReconciler) SetupWithManager(ctx context.Context, mgr
 		WithEventFilter(predicates.ResourceNotPausedAndHasFilterLabel(ctrl.LoggerFrom(ctx), r.WatchFilterValue)).
 		Build(r)
 	if err != nil {
-		return errors.Wrap(err, "failed setting up with a controller manager")
+		return errors.Wrap(err, "failed setting up with a managedController manager")
 	}
-	err = controller.Watch(
+	err = managedController.Watch(
 		&source.Kind{Type: &clusterv1.Cluster{}},
 		handler.EnqueueRequestsFromMapFunc(r.clusterToMachineHealthCheck),
 		// TODO: should this wait for Cluster.Status.InfrastructureReady similar to Infra Machine resources?
 		predicates.ClusterUnpaused(ctrl.LoggerFrom(ctx)),
 	)
 	if err != nil {
-		return errors.Wrap(err, "failed to add Watch for Clusters to controller manager")
+		return errors.Wrap(err, "failed to add Watch for Clusters to managedController manager")
 	}
 
-	r.controller = controller
-	r.recorder = mgr.GetEventRecorderFor("machinehealthcheck-controller")
+	r.controller = managedController
+	r.recorder = mgr.GetEventRecorderFor("machinehealthcheck-managedController")
```

Comment:
This is also better as just controller I think
```suggestion
	r.recorder = mgr.GetEventRecorderFor("machinehealthcheck-controller")
```